### PR TITLE
[TR] Chapter 3/1

### PR DESCRIPTION
**Pull request by Winvoker**
Related to #41

### CODE REVIEW

**Comment by Winvoker** [comment]
@lewtun

File: chapters/tr/_toctree.yml [hunk 1]
```diff
@@ -17,3 +16,20 @@
   sections:
   - local: chapter2/1
     title: Giriş
+
+- title: 3. Pretrained bir modeli fine-tune etmek
+  sections:
+  - local: chapter3/1
+    title: Giriş
+  - local: chapter3/2
```
Comment (lewtun) [inline chapters/tr/_toctree.yml:24]: We should only include the sections here that have already been translated, so everything from here and below can be removed for now :)

**Review by lewtun — COMMENTED** [review]
Thanks for this translation @Winvoker ! There's a small tweak needed to the ToC but otherwise this is looking great :)

File: chapters/tr/_toctree.yml (hunk 1 above)
Comment (Winvoker) [reply chapters/tr/_toctree.yml]: i think its done.

**Review by lewtun — APPROVED** [review]
Thank you for iterating @Winvoker - this looks great!

**Pull request merged**